src/questionnaireitem_system_viewport_size.js

```diff
@@ -0,0 +1,27 @@
+/**
+A QuestionnaireItemSystem that stores the dimension of the viewport.
+
+Reports the viewport as array [document.documentElement.clientWidth, document.documentElement.clientHeight] in pixels. It gets measured at the time of createUI().
```

Comment:
One sentence per line.

Reply:
One nitpicking I forgot: can you also update examples/developer_template.html?
The new JS file needs to be included.